pkg/share/manila/sharebackends/csicephfs.go

```diff
@@ -36,6 +38,10 @@ func (CSICephFS) BuildSource(args *BuildSourceArgs) (*v1.PersistentVolumeSource,
 		return nil, err
 	}
 
+	if args.Options.CSICEPHFSmounter != "fuse" && args.Options.CSICEPHFSmounter != "kernel" {
```

Comment:
I think default is 'fuse' as you mentioned in document, so in case of empty it should not return error.

Reply:
This is a no - `CSICEPHFSmounter` may never be empty. It must be either:
- populated by the user using the storage class parameters, or
- populated by the options parser with the default value, if the attribute has not been previously populated by the user

It's an error to have this value empty. In this case it would mean there's a bug in the parser (because it allowed an empty value here, which it shouldn't), so at least it won't go unnoticed.

Reply:
ok got that, so the options parser will always populate it with default value.